ipaplatform/base/paths.py

```diff
@@ -49,7 +49,7 @@ class BasePathNamespace(object):
     HTTPD_IPA_CONF = "/etc/httpd/conf.d/ipa.conf"
     HTTPD_NSS_CONF = "/etc/httpd/conf.d/nss.conf"
     HTTPD_SSL_CONF = "/etc/httpd/conf.d/ssl.conf"
-    IPA_KEYTAB = "/etc/httpd/conf/ipa.keytab"
+    HTTP_KEYTAB = "/etc/gssproxy/http.keytab"
```

Comment:
There is none, once upgrade will be handle we'll mv and chown/chmod/chcon the file in the right place.